kin-sdk/src/test/java/kin/sdk/TransactionSenderTest.java

```diff
@@ -134,6 +133,28 @@ public void sendTransaction_ToAccountNotExist() throws Exception {
         transactionSender.sendTransaction(transaction);
     }
 
+	@Test
+	public void sendTransaction_Http_307_Response_Success() throws Exception {
+		//send transaction fetch first to account details, then from account details, and finally perform tx which will return 307
+		// and then 200. here we mock all 4 responses from server to achieve success operation
+		mockWebServer.enqueue(TestUtils.generateSuccessMockResponse(this.getClass(), "tx_account_from.json"));
+		mockWebServer.enqueue(TestUtils.generateSuccessMockResponse(this.getClass(), "tx_account_to.json"));
+		// No need for a real location because any way it is local host
+		mockWebServer.enqueue(TestUtils.generateSuccessHttp307MockResponse(this.getClass(), "tx_success_res.json", ""));
```

Comment:
similar to the comment on `ServerTest`, the first 307 response shouldn't include a body

Reply:
fixed